<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>-5</x>
      <y>1</y>
      <width>461</width>
      <height>441</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Club</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Role</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Goals</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Penalty</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="topScorersTable">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>461</width>
      <height>441</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Club</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Role</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Goals</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Penalty</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="topPenaltyTable">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>461</width>
      <height>441</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Club</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Role</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Goals</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Penalty</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="WorstscorersTable">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>481</width>
      <height>441</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Club</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Role</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Goals</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Penalty</string>
     </property>
    </column>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>0</y>
      <width>171</width>
      <height>271</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="openfilebut">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Open File</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>131</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Display top scorers</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hidetopscorrers">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>131</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Hide top scorers</string>
     </property>
    </widget>
    <widget class="QPushButton" name="biggestpenalty">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>131</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Top penalty</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hidetoppenalty">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>131</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Hide top penalty</string>
     </property>
    </widget>
    <widget class="QPushButton" name="worstscorersbut">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>141</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Display worst scorers</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hideworstscorers">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>141</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Hide top penalty</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addbut">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>141</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Player</string>
     </property>
    </widget>
    <widget class="QPushButton" name="editbut">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>141</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Edit selected player</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="EDITMENU">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>270</y>
      <width>181</width>
      <height>231</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="nameline">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="clubline">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="goalsline">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="penaltyline">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="roleline">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter club:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter goals:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter penalty:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter role:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>EDIT MENU</string>
     </property>
    </widget>
    <widget class="QPushButton" name="confirmbut">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
    <widget class="QPushButton" name="confirmbut_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
